doc/server-secrets.md

```diff
@@ -244,6 +244,17 @@ Create an account, sign in and obtain a uuid and token from your
 to give your self-hosted Shields installation access to a
 private SonarQube instance or private project on a public instance.
 
+### StackApps
```

Comment:
Something I've often wondered about is how cognitively transferable one Stack* is to another for the average dev. I.e. is someone looking at this from a purely StackOverflow-centric frame of mind going to know to look/search for StackApps or StackExchange?

Reply:
`StackApps` is where we keep track of API registration. It's not really meaningful to a typical user.  Stack Overflow or Stack Exchange Network would be more recognizable names.

Reply:
I've tweaked the header to include all 3